deps/verifier/Cargo.toml

```diff
@@ -4,31 +4,44 @@ version = "0.1.0"
 edition = "2021"
 
 [features]
-default = [ "all-verifier" ]
-all-verifier = [ "tdx-verifier", "sgx-verifier", "snp-verifier", "az-snp-vtpm-verifier", "az-tdx-vtpm-verifier", "csv-verifier", "cca-verifier", "se-verifier" ]
-tdx-verifier = [ "eventlog-rs", "scroll", "intel-tee-quote-verification-rs" ]
-sgx-verifier = [ "scroll", "intel-tee-quote-verification-rs" ]
-az-snp-vtpm-verifier = [ "az-snp-vtpm", "sev", "snp-verifier" ]
-az-tdx-vtpm-verifier = [ "az-tdx-vtpm", "openssl", "tdx-verifier" ]
-snp-verifier = [ "asn1-rs", "openssl", "sev", "x509-parser" ]
-csv-verifier = [ "openssl", "csv-rs", "codicon" ]
-cca-verifier = [ "ear", "jsonwebtoken", "veraison-apiclient" ]
-se-verifier = [ "openssl", "pv", "serde_with", "tokio/sync" ]
+default = ["all-verifier"]
+all-verifier = [
+    "tdx-verifier",
+    "sgx-verifier",
+    "snp-verifier",
+    "az-snp-vtpm-verifier",
+    "az-tdx-vtpm-verifier",
+    "csv-verifier",
+    "cca-verifier",
+    "se-verifier",
+]
+tdx-verifier = ["eventlog-rs", "scroll", "intel-tee-quote-verification-rs"]
+sgx-verifier = ["scroll", "intel-tee-quote-verification-rs"]
+az-snp-vtpm-verifier = ["az-snp-vtpm", "sev", "snp-verifier"]
+az-tdx-vtpm-verifier = ["az-tdx-vtpm", "openssl", "tdx-verifier"]
+snp-verifier = ["asn1-rs", "openssl", "sev", "x509-parser"]
+csv-verifier = ["openssl", "csv-rs", "codicon"]
+cca-verifier = ["ear", "jsonwebtoken", "veraison-apiclient"]
+se-verifier = ["openssl", "pv", "serde_with", "tokio/sync"]
 
 [dependencies]
 anyhow.workspace = true
 thiserror.workspace = true
 asn1-rs = { version = "0.5.1", optional = true }
 async-trait.workspace = true
-az-snp-vtpm = { version = "0.5.3", default-features = false, features = ["verifier"], optional = true }
-az-tdx-vtpm = { version = "0.5.3", default-features = false, features = ["verifier"], optional = true }
+az-snp-vtpm = { version = "0.5.3", default-features = false, features = [
+    "verifier",
+], optional = true }
+az-tdx-vtpm = { version = "0.5.3", default-features = false, features = [
+    "verifier",
+], optional = true }
 base64 = "0.21"
 bincode = "1.3.3"
 byteorder = "1"
 cfg-if = "1.0.0"
 codicon = { version = "3.0", optional = true }
 # TODO: change it to "0.1", once released.
-csv-rs = { git = "https://github.com/openanolis/csv-rs", rev = "b74aa8c", optional = true }
+csv-rs = { version = "=0.1.0", git = "https://github.com/openanolis/csv-rs", rev = "b74aa8c", optional = true }
```

Comment:
is this intentionally both version and ref?

Reply:
I am not sure if it is a bug in Cargo, or something else, but if I didn't enforce the specific version of the crate, it was being mangled on re-build.